src/kirby/components/button/button.component.scss

```diff
@@ -28,9 +28,9 @@
   }
 
   &[colortype='secondary'] {
-    background-color: get-color('kirby-secondary');
-    border-color: get-color('kirby-secondary');
-    color: get-color('kirby-secondary-contrast');
+    background-color: get-color('kirby-contrast-light');
```

Comment:
Shouldn't this be `kirby-secondary`? I know that we have a button with a role of secondary, but it's a bit confusing...

Reply:
Agreed - I suggest we do another overhaul of the button colors after a discussion with the UX folks.